src/Plugin/Field/FieldWidget/TideSiteRestrictionFieldWidget.php

```diff
@@ -141,8 +142,8 @@ protected function userOptionsFilter(AccountProxyInterface $account, array $opti
   public function massageFormValues(array $values, array $form, FormStateInterface $form_state) {
     $results = parent::massageFormValues($values, $form, $form_state);
     $options = [];
-    $entity = $form_state->getFormObject()->getEntity();
-    if (!$this->helper->canBypassRestriction($this->currentUser)) {
+    if (!$this->helper->canBypassRestriction($this->currentUser) && $form_state->getFormObject() instanceof EntityFormInterface) {
```

Comment:
I'm just trying to understand the logic here, why should this statement only run if the user can't bypass the site restriction?